api/controllers/MerchStoreController.ts

```diff
@@ -188,4 +193,44 @@ export class MerchStoreController {
     await this.merchStoreService.updateOrderItems(fulfillOrderRequest.items);
     return { error: null };
   }
+
+  @Get('/pickupEvent/past')
```

Comment:
> I could imagine reasons to look at past event events because a member want want to see something related to the event, but I can't think of a good reason to scroll through past pickup events.

The reason we have `/past` as well is so that at the end of the quarter, an admin can look at all the unfulfilled orders from past events and take action on those. This route would serve that functionality, instead of having something like `GET /order/past` with query params or something to get past orders that weren't fulfilled since I think orders temporally are tied to pickup events.

Reply:
> Should this be under the /order prefix instead? Also nit but thoughts on /pickup instead of /pickupEvent? It just looks a little neater but I also prefer /first-second instead of /firstSecond because it's a little more readable when looking at requests.

Good call. I like the idea of `/order/pickup` since pickup events are entities that are tightly related to orders.

Reply:
> The reason we have `/past` as well is so that at the end of the quarter, an admin can look at all the unfulfilled orders from past events and take action on those. In our case, membership will cancel/refund all orders that haven't been fulfilled by the end of the quarter. This route would serve that functionality, instead of having something like `GET /order/past` with query params or something to get past orders that weren't fulfilled since I think orders temporally are tied to pickup events.

What about something like `/order/unfulfilled` then? That seems more accurate for what the route's intended for. The route would return unfulfilled orders, "unfulfilled" meaning orders with past pickup events that weren't marked as completed.

Reply:
We're adding a `status` field to orders where `status` is one of `{PLACED, MISSED, CANCELLED, FULFILLED}` with query params added for `status` in `GET /order` so I think we can use that for getting all missed orders and taking action. I can remove the `/past` endpoint